test/acceptance/tests.py

```diff
@@ -1102,6 +1105,43 @@ def test_train_self_peer_staff(self, peer_grades_me):
             ])
 
 
+@ddt.ddt
+class MultipleOpenAssessmentTest(OpenAssessmentTest, FullWorkflowMixin):
+    """
+    Test the multiple peer-assessment flow.
+    """
+
+    def setUp(self):
+        super(MultipleOpenAssessmentTest, self).setUp('multiple_ora')
+        self.staff_area_page = StaffAreaPage(self.browser, self.problem_loc)
+
+    def set_vert_no(self, vert_no):
```

Comment:
Don't duplicate this method in the a11y test file and here. Create a mixin that can be shared.